index.html

```diff
@@ -66,8 +66,210 @@ <h1 id="main-header">I’m Alexandra.
+            <p class="info-text" id="about-text">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <div class="link" id="about-myself-link">
+                LET'S CONNECT
+            </div>
+
+            <div id="social-links" id="about-myself-social-links">
+                <ul id="social-links-container">
+                    <li><a href=""><img src="./Images/twitter-logo.png" class="social-links-img" alt=""> </a></li>
+                    <li>
+                        <a href="">
+                            <span id="linkedin-container">
+                                    <img src="./Images/o-png.png" alt="" id="o-png">
+                                    <img src="./Images/I-png.png" alt="" id="I-png"> 
+                                    <img src="./Images/linkedin-logo.png" class="social-links-img" alt="">
+                            </span>
+                        </a>
+                    </li>
+                    <li><a href=""><img src="./Images/M-logo.png" class="social-links-img" alt=""></a></li>
+                    <li><a href=""><img src="./Images/github-logo.png" class="social-links-img" alt=""></a></li>
+                    <li><a href=""> <img src="./Images/peace-sign.png" class="social-links-img" alt=""></a></li>
+                </ul>
+          </div>
+
+          <div class="view-project-button">
+            <button>Get my resume</button>
+          </div>
+
+        </section>
+
+        <!-- ABOUT MYSELF SECTION END  -->
+
+        <!-- LANGUAGE SECITON START  -->
+        <ul id="language-section">
+            <li id="language-header-box">
+                <div id="language-header">
+                    <h2>Languages</h2>
+                    <i class="fa-solid fa-chevron-down"></i>
+                </div>
+
+                <ul id="skills-container-box">
+                    <li class="skills">
+                        <img src="./Images/javascript-img.png" alt="">
+                        <span>JavaScript</span>
+                    </li>
+    
+                    <li class="skills">
+                        <img src="Images/html-img.png" alt="">
+                        <span>HTML</span>
+                    </li>
+    
+                    <li class="skills">
+                        <img src="Images/css-img.png" alt="">
+                        <span>CSS</span>
+                    </li>
+                </ul>
+            </li>
+            
+            <li class="framework-skill" id="framework-container"> 
+                <h2 id="framework-skill-header">Frameworks</h2>
+                <img src="./Images/right.png" alt="">
+            </li>
+
+            <img src="Images/Separator Bottom.png" alt="" id="separator-img">
+
+            <li class="framework-skill">
+
+                <h2>Skills</h2>
```

Comment:
- [ ] Kindly inclose `language-section` `ul` inside `about-myself` section as it is part of that section.

Reply:
@shella12  Thank you for all the clear instructions.